website/docs/en/plugins/webpack/_meta.json

```diff
@@ -1,21 +1,21 @@
 [
   "index",
-  "entry-plugin",
+  "internal-plugins",
```

Comment:
The internal plugins are usually not important for our users, can we move the `internal-plugins` to the bottom of the list?